### PR TITLE
GG-22666 Add DR monitoring and management capabilities to control.sh

### PR DESCRIPTION
**Pull request by ibessonov**

### CODE REVIEW

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/dr/subcommands/DrAbstractSubCommand.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            log.severe(CommandLogger.errorMessage(e));
+
+            throw e;
+        }
+
+        return null;
+    }
+
+    /** {@inheritDoc} */
+    protected VisorResultDto execute0(
```
Comment (akuznetsov-gridgain) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/dr/subcommands/DrAbstractSubCommand.java]: Missing `@Override`;

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/dr/subcommands/DrCacheCommand.java [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+            }
+
+            return null;
+        }
+    }
+
+    /** */
+    public static class DrCacheArguments implements DrAbstractSubCommand.Arguments<VisorDrCacheTaskArgs> {
+        private final String regex;
+        private final boolean config;
```
Comment (akuznetsov-gridgain) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/dr/subcommands/DrCacheCommand.java]: Missing `/** */` for fields.

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/dr/subcommands/DrCacheCommand.java [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+        SenderGroup senderGroup = SenderGroup.ALL;
+        String senderGroupName = null;
+        Action action = null;
+
+        String nextArg;
+
+        //noinspection LabeledStatement
+        args_loop: while ((nextArg = argIter.peekNextArg()) != null) {
+            switch (nextArg.toLowerCase(Locale.ENGLISH)) {
+                case "--config":
```
Comment (akuznetsov-gridgain) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/dr/subcommands/DrCacheCommand.java]: May be it make sense to declare static constants for all strings like "--config" in order to have them in one place.

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/dr/subcommands/DrAbstractSubCommand.java (hunk 1 above)
Comment (ibessonov) [reply modules/core/src/main/java/org/apache/ignite/internal/commandline/dr/subcommands/DrAbstractSubCommand.java]: fixed

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/dr/subcommands/DrCacheCommand.java (hunk 3 above)
Comment (ibessonov) [reply modules/core/src/main/java/org/apache/ignite/internal/commandline/dr/subcommands/DrCacheCommand.java]: fixed

**Pull request closed without merging**